packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -260,6 +252,170 @@ const EVENTS = {
     group: GROUPS.thirdPartyAuth,
     event: 'view',
   },
+
+  /* Everything under this point should be Settings events, aka 'fxa_pref' group */
+  // temp fallback tests
```

Comment:
Yeah we probably do need an issue filed for that, it's mostly just been hanging out in my brain since I'm actively working on something related to this (grooming stuff in [FXA-999](https://mozilla-hub.atlassian.net/browse/FXA-999) / working through the spike in there) and figured as part of wrapping that up I'd remove it but not sure that timing is going to line up perfectly.